packages/exchange-io-erc1888/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@energyweb/exchange-io-erc1888",
+  "version": "1.0.0",
+  "description": "ERC1888 withdwaral/deposit processing for exchange",
+  "author": "EnergyWeb DevHub GmbH; Aleksandr Marenin, aleksandr.amarenin@energyweb.com",
```

Comment:
Is the email correct? :)

Reply:
fixed ))

Reply:
I think it's still incorrect. Should be @enerygweb.org instead of @energyweb.com

Reply:
🤦 Thanks, now it is fixed